.github/workflows/dockerfile_push.yaml

```diff
@@ -0,0 +1,32 @@
+name: Push Dockerfile to Quay
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'Dockerfile'
+jobs:
+  push-dockerfile:
+    name: Push dockerfile
+    runs-on: ubuntu-20.04
+    env:
+      QUAY_ORG: hacbs-release
+      QUAY_REPO: release-utils
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_ROBOT_USER }}
+          password: ${{ secrets.QUAY_ROBOT_TOKEN }}
+      - name: Build container and push it to Quay
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          file: Dockerfile
+          tags: |
+            quay.io/${{ env.QUAY_ORG }}/${{ env.QUAY_REPO }}:latest
```

Comment:
We should move away from using `latest`. Can we change it to `main`?

Reply:
done